assets/js/payment-method-extensions/payment-methods/stripe/credit-card/index.js

```diff
@@ -32,18 +34,16 @@ const StripeComponent = ( props ) => {
 	return <StripeCreditCard stripe={ stripePromise } { ...props } />;
 };
 
-const StripeLabel = ( { components = {} } ) => {
-	const { PaymentMethodIcons } = components;
-
-	if ( ! PaymentMethodIcons || cardIcons.length === 0 ) {
-		return __( 'Credit/Debit Card', 'woo-gutenberg-products-block' );
-	}
-
-	return <PaymentMethodIcons icons={ cardIcons } />;
+const StripeLabel = () => {
+	return (
+		<span className="payment-method-label-with-icon">
```

Comment:
We need to style these differently than tabs with just icons, no text. I added some CSS rules for this so there is correct spacing.